### DISCUSSION TITLE
QR code scanning detection area only on top right of the screen

### DISCUSSION THREAD

**Discussion by julienlr** [capawesome-team/capacitor-mlkit#183] [Q&A]
Hello, I wanted to know if someone already had this case :  
With a large screen Android phone, when I launch the BarcodeScanner to scan a QR Code, it detects only qr code on top right of the screen.
I tried with another Android device, smaller, and I can scan anywhere on the screen.
Does anyone would have a solution ? 
Thanks in advance

**Comment by robingenz** [comment]
Which plugin version do you use?

**Reply by julienlr** [reply]
i am using 5.4.0

**Reply by robingenz** [reply]
Okay, the version should work. Unfortunately, I don't have an Android device with a large screen. Feel free to debug it yourself and create a PR.
